website/docs/reference/dbt-jinja-functions/source.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "source"
 id: "source"
-description: "Returns a relation & creates dependencies between source and model."
+description: "Read this guide to understand the souce Jinga function in dbt."
```

Comment:
`Jinga` should be `Jinja`

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/source

content="Read this guide to understand the source Jinja function in dbt."